src/ProjectTemplates/Web.ProjectTemplates/content/Components-CSharp/.template.config/ide.host.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json.schemastore.org/ide.host",
+  "order": 610,
+  "icon": "icon.png",
+  "disableHttpsSymbol": "NoHttps",
+  "symbolInfo": [
+    {
+      "id": "UseServer",
+      "isVisible": false,
+      "persistenceScope": "templateGroup"
+    },
+    {
+      "id": "UseClient",
+      "isVisible": false,
+      "persistenceScope": "templateGroup"
+    },
+    {
+      "id": "PWA",
+      "isVisible": false,
```

Comment:
We don't want to enable choosing PWA or client/server right now?

Reply:
PWA is only for when you have enabled webassembly. Until we don't have support for it, we won't turn on the flag